main.go

```diff
@@ -39,7 +31,7 @@ func main() {
 		Debug:   debug,
 	}
 
-	err := providerserver.Serve(context.Background(), provider.New(version), opts)
+	err := providerserver.Serve(context.Background(), provider.New(version2.ProviderVersion), opts)
```

Comment:
ASK: Why not `version` but `version2`?

Reply:
This was unnecessary and has been fixed!